unstructured/file_utils/filetype.py

```diff
@@ -347,7 +368,17 @@ def _is_text_file_a_json(
         with open(filename) as f:
             file_text = f.read()
 
-    return re.match(LIST_OF_DICTS_PATTERN, file_text) is not None
+    lines = file_text.strip().splitlines()
```

Comment:
In case the file is really big, it might be good to just read the first 10 or 20 lines of the file instead of the entire file.